docs/data/date-pickers/custom-components/PickerDay2Demo.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { ThemeProvider, createTheme, useTheme } from '@mui/material/styles';
+import Box from '@mui/material/Box';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { StaticDateRangePicker } from '@mui/x-date-pickers-pro/StaticDateRangePicker';
+import { DateRangePickerDay2 } from '@mui/x-date-pickers-pro/DateRangePickerDay2';
+import { getEnhancedCustomTheme } from './getEnhancedCustomTheme';
+
+export default function PickerDay2Demo() {
+  const currentTheme = useTheme();
+  const theme = createTheme(getEnhancedCustomTheme(currentTheme.palette.mode));
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <Box sx={{ display: 'flex', gap: 4 }}>
+        <StaticDateRangePicker
```

Comment:
We should probably have one `StaticDatePicker` and one `StaticDateRangePicker` in this demo instead of 2 `StaticDateRangePicker`.

And then have a 2nd demo that shows the usage of CSS variables (I would use a `StaticDatePicker` there as well to keep it as simple as possible)

Reply:
Done! I separated the demo with the custom theme entirely - I think it's valuable to have a customization example like that in this case. LMK what you think